src/anet.c

```diff
@@ -428,7 +433,12 @@ int anetWrite(int fd, char *buf, int count)
     while(totlen != count) {
         nwritten = write(fd,buf,count-totlen);
         if (nwritten == 0) return totlen;
-        if (nwritten == -1) return -1;
+        if (nwritten == -1)
+        {
+            if(errno==EINTR)
+                continue;
+            return -1;
+        }
```

Comment:
This is dead now and removed here: f657315e601d28542a1f8d79ebdc1105bf803341
No need anymore.